biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/solar-panel.ts

```diff
@@ -0,0 +1,41 @@
+import { getSunlight } from '@biogrid/weather';
+import {EnergySource} from './bioenergy-source'
+import {
+  Validatable,
+  validate,
+  Power,
+  SunlightIntensity,
+} from '@biogrid/grid-simulator';
+
+export class SolarPanel extends EnergySource {
+  private sizeSqMtr: number;
+
+  constructor(sizeSqMtr: number, efficiency = 0.175, longitude = 0, latitude = 0) {
```

Comment:
nit: what is this default 0.175? can we lave a comment explaining where that comes from